content/post/2019-08-31-tests-as-linear.Rmd

```diff
@@ -0,0 +1,1458 @@
+
+<!-- From https://stackoverflow.com/a/37839683/1297830 -->
+<link rel="stylesheet" type="text/css" href="include/hideOutput.css">
+<script src="include/hideOutput.js"></script>
+
+作者：Jonas Kristoffer Lindeløv ([博客](https://lindeloev.net), [个人主页](http://personprofil.aau.dk/117060)).<br />
+最后更新日期：`r format(Sys.time(), '%d %B, %Y')` (见[更新日志](https://github.com/lindeloev/tests-as-linear/commits/master))。<br />
+可查阅[Python 版本](https://eigenfoo.xyz/tests-as-linear/) 和[Twitter 概况](https://twitter.com/jonaslindeloev/status/1110907133833502721).
+
+<br />
+
+```{r echo=FALSE}
+# Options for building this document
+knitr::opts_chunk$set(
+  dev = "svg",
+  fig.height = 4,
+  fig.width = 6,
+  fig.align = "center",
+  message = FALSE,
+  warning = FALSE
+)
+
+# To show tables.
+print_df <- function(D,
+                     decimals = 4,
+                     navigate = FALSE) {
+  DT::datatable(
+    mutate_if(D, is.numeric, round, decimals),
+    rownames = FALSE,
+    options = list(
+      searching = FALSE,
+      lengthChange = FALSE,
+      ordering = FALSE,
+      autoWidth = TRUE,
+      bPaginate = navigate,
+      bInfo = navigate,
+      paging = navigate
+    )
+  )
+}
+```
+
+> 译者注：
+>
+> **翻译说明：本文翻译自英文文章[Common statistical tests are linear models (or: how to teach stats)](https://lindeloev.github.io/tests-as-linear/)，作者 Jonas Kristoffer Lindeløv；本翻译已获得作者授权，首发于[统计之都](https://cosx.org)，译者黄俊文。**<br/>
+> **This Chinese article is translated from English version [Common statistical tests are linear models (or: how to teach stats)](https://lindeloev.github.io/tests-as-linear/) written by Jonas Kristoffer Lindeløv. And the translation is authorized by original author, firstly published on [Capital of Statistics](https://cosx.org).**
+>
+> 译者评语：本文使用了统一的线性模型框架，来解释各种统计检验，极富思考性和启发性。
+> 
+> 翻译准则：除了众所周知的（“牛顿”、“贝叶斯”），基本上不翻译人名。其余如 ANOVA 或 goodness of fit 等中文统计教材有常见叫法的，翻译成（除人名部分外）中文版本。除了 markdown 语法和代码和英文文本片段之外，使用正规中文标点符号。本文译者水平有限，欢迎各位在中文版 GitHub repo 下提出对各种错漏的修改建议。
+>
+> 运行：要运行本文所有代码，你需要安装以下软件包：
+> ```r
+> pkgs = c(
+>     'car',
+>     'broom',
+>     'DT',
+>     'knitr',
+>     'tidyverse',
+>     'sm',
+>     'rprojroot',
+>     'remotes'
+> )
+> install.packages(pkgs)
+> 
+> remotes::install_github("jumpingrivers/headR")
+> remotes::install_github("thomasp85/patchwork")
+> ```
+
+此文档整理如下表。它展示了在常见的参数和“非参”检验之下的线性模型。在同一个框架下展示所有检验，表达出了它们之间的许多相似之处。获得[图片版本](linear_tests_cheat_sheet.png)或 [PDF 版本](linear_tests_cheat_sheet.pdf)或[在线版](linear_tests_cheat_sheet.html)。
+
+***
+
+[![](linear_tests_cheat_sheet.png)](linear_tests_cheat_sheet.pdf)
+
+***
+
+
+# 常见检验的简洁本质
+
+大部分常见统计模型（t 检验、相关性、方差分析（ANOVA）、卡方，等等）都是线性模型的特殊形式，或者是很接近的近似行是。这漂亮的简洁性，意味着我们需要学习更少东西。特别地，这都来源于大部分学生从高中就学习的模型：$y = a \cdot x + b$。不幸地，统计入门课程通常把各种检验教导魏独立的工具，更不要提它们对学生和教师生活所增加的复杂性了。
+
+当学生想分别学习每一个检验背后的参数假设的时候，相比于从线性模型进行推导，教材里多余的复杂性变得更为复杂。
+
+因此，我认为，先教导线性模型和扩展，*然后*对于一些特殊形式进行“改名”，这种方式是一种优秀的教导策略，并且有助于更深刻地*理解*假设检验。因为线性模型在频率学派、贝叶斯学派、基于置换的推断之间，都是相同的。我认为从模型开始，比从 p 值、第 1 类错误、贝叶斯因子、或其它东西开始，更为友好。
```

Comment:
“基于置换的推断” 改为 “自助法统计推断”

Reply:
这里应该是基于置换的检验推断，毕竟后文提到了 U 检验， 自助法的话文章中应该提到重抽样 resampling 或者 bootstrap 之类的字眼，这里原文是

Since linear models are the same across frequentist, Bayesian, and **permutation-based** inferences